src/client/components/dimension-actions-menu/dimension-actions-menu.tsx

```diff
@@ -89,7 +89,7 @@ export const DimensionActions: React.SFC<DimensionActionsProps> = (props: Dimens
   }
 
   function onPin() {
-    if (isStringDimension) clicker.pin(dimension);
+    if (isPinable) clicker.pin(dimension);
```

Comment:
Could we move decision about "pinability" into clicker.pin method?

Reply:
Maybe both? In `Clicker.pin` we could throw error and treat it as unexpected exception. Here, this check is for safety if user clicks in some weird moment on disabled button. We should there silently fail.

I wonder if such weird moment could even exist ...

Reply:
Damn, this is not a button but icon! So we visually show it is disabled in render code, but also we need this check here, because click event still goes through. Bad accessibility, but I'd like to tackle this another time.